package.json

```diff
@@ -16,10 +16,13 @@
   "dependencies": {
     "@(-.-)/env": "0.3.2",
     "@keyv/sqlite": "3.6.5",
+    "cheerio": "1.0.0-rc.12",
     "discord.js": "14.11.0",
     "dotenv": "16.0.3",
     "glob": "^10.2.4",
     "keyv": "4.5.2",
+    "pnpm": "8.5.1",
+    "tsc": "2.0.4",
```

Comment:
these are unused krub, please remove (and update lockfile)

Reply:
✅